crates/uv/tests/pip_compile.rs

```diff
@@ -2247,7 +2228,7 @@ fn compile_wheel_path_dependency() -> Result<()> {
         # via flask
     click==8.1.7
         # via flask
-    flask @ /[TEMP_DIR]/
+    flask @ [TEMP_DIR]/flask-3.0.0-py3-none-any.whl
```

Comment:
We don't replace the entire path anymore, just the non-deterministic part. This should generally improve test coverage and clarity.